test/compilable/testcolor.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+set -eo pipefail
```

Comment:
Technically we don't need this, because it's the default of the Bash environment in which the testsuite is run.
Check e.g. https://github.com/dlang/dmd/blob/master/test/tools/sh_do_test.sh

Reply:
That's good, but test scripts might as well behave the same whether they're being run from within the harness or directly.

Reply:
FYI: One can't run most of the test scripts directly because they depend on common variables sourced by this environment, but you can always do e.g. `./run.d compilable/testcolor.sh` to run them directly.